mxcube3/routes/Login.py

```diff
@@ -49,7 +49,10 @@ def login():
 #        "local_contact": self.get_session_local_contact(todays_session['session']['sessionId']),
 #        "person": prop['Person'],
 #        "laboratory": prop['Laboratory']}
-        mxcube.queue = jsonpickle.decode(session.get("queueList"))
+	try:
+            mxcube.queue = jsonpickle.decode(session.get("queueList"))
```

Comment:
Wouldn't it be enough to do like this:

```python
mxcube.queue = jsonpickle.decode(session.get("queueList","{}"))
```
instead of raising the exception ?

I guess the flask session obeys the `dict` semantics